src/puyapy-stubs/op.pyi

```diff
@@ -2142,56 +2142,140 @@ class EllipticCurve:
 class Global:
     """Functions for the op: `global`"""
 
-    @staticmethod
-    def min_txn_fee() -> UInt64:
-        """
-        global field F
+    min_txn_fee: UInt64 = ...
```

Comment:
Can we annotate all these with `ClassVar`?

Reply:
Also, I don't suppose there's any way we can mark these as readonly... I know you mentioned that having this as a type and then an instance so that properties are usable didn't work with the doc generation :(

Reply:
Done